src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -72,9 +86,12 @@
 
     @Test
     public void getStatisticAboutBananaRepeatedMethodCall() {
+        try {
         workWithFile.getStatistic("banana.csv", BANANA_RESULT_FILE);
         workWithFile.getStatistic("banana.csv", BANANA_RESULT_FILE);
-
+        } catch (FileNotFoundException e) {
```

Comment:
Catching `FileNotFoundException` is not appropriate here because the `getStatistic` method can throw `IOException`. Consider catching `IOException` instead.